clifford/cga.py

```diff
@@ -164,11 +164,9 @@ def __init__(self,cga, *args):
             
         # from vectors on flat         
         else:
-            nulls = map(self.cga.null_vector,args)
-            if self.einf not in nulls:
-                nulls = list(nulls)+[self.einf]
-                
-            self.mv = reduce(op,nulls) 
+            nulls = list(map(self.cga.null_vector,args))
```

Comment:
Is it sufficient to change only this line? What is the effect of changing the other lines? Is it possible that `op(self.mv, self.einf)  == 0` but `self.einf not in nulls`?

Reply:
Makes sense to me, thanks for the summary.